_posts/2022-05-06-grants-bitcoin-open-source.markdown

```diff
@@ -0,0 +1,41 @@
+
+## How much to ask for
+
+The round number people tend to use is $100,000. This simplifies the process for grantors, so they don't have to make judgment calls on your standard of living or life circumstances. Some programs will adjust their grants based on need. $100,000 in San Francisco is very different than $100,000 in San Salvador. This requires more effort from both the grantor and the grantee to find the number that makes sense. It is also standard for most programs to allow you to take grants from other sources, but some require exclusivity. This is to prevent situations where people have stacked grants to the point they are making a higher than industry salary, which can appear greedy. I think you are usually fine if you are as transparent as possible with your needs and where you expect to source money.
+
+## Sustainability
+
+The goal of these grant programs is not to make you rich. If you are looking for financial upside, going the grant program route isn't for you. The list remains short of full-time roles for bitcoin open-source work, but they do exist. Grants are typically 12-month terms, but it is fair to ask about grant length and the renewal process. The majority of the grant turnover I see is not driven from the grantor side -- usually, either the grantee quits to do something else or they move to another sponsor because of better terms or to associate with a grant program with a better reputation.
+
+## Association with Grantors
+
+There is some level of prestige among the different programs. Most of this is based on the sponsor's standing in the community. Jonas Schnelli got [some flak](https://twitter.com/_jonasschnelli_/status/1360259171036332036) for taking money from Marathon Mining when their company policy was to [censor transactions](https://www.coindesk.com/tech/2021/05/07/marathon-miners-have-started-censoring-bitcoin-transactions-heres-what-that-means/). For me, there are two ways to look at this: 1) By taking money from a source, it could be interpreted as an implicit endorsment of their business and role in the community or 2) taking money from a less-than-ideal source could be seen as reparations or their attempt at gaining goodwill with the community with money. In other words, taking tainted money and turning it into good.
+
+Obviously, it's not so black and white, but if you are foregoing financial upside to work on open-source, you probably want to think about where you fall here.
+
+## BTC or USD
+
+Some programs offer grants either dominated in fiat or bitcoin. This can go very wrong. We are all here because we believe in bitcoin and have high hopes for its future, but bitcoin hasn't yet arrived. If you can't cover your bills because of a price crash, now you are back to worrying about funding. Please don't be irresponsible with philanthropic money given to you to create public good.
```

Comment:
```suggestion
Some programs offer grants either dominated in fiat, bitcoin, or both. This can go very wrong. We are all here because we believe in bitcoin and have high hopes for its future, but bitcoin hasn't yet arrived. If you can't cover your bills because of a price crash, now you are back to worrying about funding. Please don't be irresponsible with philanthropic money given to you to create public good.
```

Reply:
"This can go very wrong", was going to suggest that taking a bitcoin denominated grant is explicitly what carries risk. But, fiat carries immense risk as well. In general, I think the idea is captured in its current form.